src/main/java/core/basesyntax/CompareTwoAuto.java

```diff
@@ -1,9 +1,15 @@
 package core.basesyntax;
 
+import java.util.function.BinaryOperator;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для сравнения мощности
  * двигателей у двух авто (enginePower) и возврата разницы между ними.
  * Метод должен принимать Integer.</p>
  */
-public class CompareTwoAuto {
+public class CompareTwoAuto implements BinaryOperator<Integer> {
+    @Override
+    public Integer apply(Integer enginePower, Integer enginePower2) {
+        return enginePower - enginePower2;
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/code_style_requirements.html

please see the very first point here
also, it is recommended to read the whole doc

Reply:
Done